core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -119,36 +122,51 @@ var ExplorationEditorFeedbackTab = function() {
       `${suggestionRowClassName} ${feedbackSubjectClassName}`,
       suggestionDescription));
     expect(await matchingRow.isDisplayed()).toBe(true);
-    await matchingRow.click();
+    await action.click('Matching Row', matchingRow);
     expect(await viewSuggestionButton.isDisplayed()).toBe(true);
-    await viewSuggestionButton.click();
+    await action.click('View Suggestion Button', viewSuggestionButton);
     expect(await rejectSuggestionButton.isDisplayed()).toBe(true);
-    await suggestionReviewMessageInput.sendKeys('Review message');
-    await rejectSuggestionButton.click();
+    await action.sendKeys(
+      'Suggestion Review Message Input',
+      suggestionReviewMessageInput, 'Review message');
+    await action.click('Reject Suggestion Button', rejectSuggestionButton);
     await waitFor.invisibilityOf(
       acceptSuggestionButton, 'Suggestion modal takes too long to disappear');
     await waitFor.pageToFullyLoad();
   };
 
   this.selectLatestFeedbackThread = async function() {
     await waitFor.visibilityOf(
-      await element.all(by.css(suggestionRowClassName)).first(),
+      await element.all(
+        by.css(suggestionRowClassName)).first(),
       'No feedback messages are visible.');
-    await element.all(by.css(suggestionRowClassName)).first().click();
+    var suggestionRowClassNameFirst =
+      element.all(by.css(suggestionRowClassName)).first();
+    await action.click(
+      'Suggestion Row Class Name', suggestionRowClassNameFirst);
```

Comment:
These are just suggestionRow elements, not class names. We are just using the class name in suggestionRowClassName to find the suggestionRow element